Monika After Story/game/script-greetings.rpy

```diff
@@ -773,6 +773,30 @@ label greeting_latin:
     m 3eua "If you're wondering about what I said, it's simply 'We meet again! How are you?'"
     return
 
+#TODO needs additional dialogue so can be used for all aff
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_esperanto",
+            unlocked=True,
+            aff_range=(mas_aff.NORMAL, None),
+        ),
+        code="GRE"
+)
+
+label greeting_esperanto:
+#    m "Mia kusenveturilo estas plena je angiloj."  # execute obligatory monty.py
+    m 1eub "Saluton, mia kara [player]."
+    m 5eub "Kiel vi fartas?"
+    m 2efb "Ĉu estas vi preta por la tago?"
+    m "Are you ready to seize the day?"
+    m 4hub "That was just a bit of Esperanto. It's a language that was created artificially instead of having evolved naturally."
```

Comment:
```suggestion
    m 3eud "That was just a bit of Esperanto...{w=0.5}a language that was created artificially instead of having evolved naturally."
```